src/com_tjnotifications/admin/views/logs/tmpl/default.php

```diff
@@ -0,0 +1,198 @@
+						</th>
+						<th class="minwidth-150px">
+							<?php echo JHtml::_('grid.sort', Text::_("COM_TJNOTIFICATIONS_VIEW_NOTIFICATIONS_DEFAULT_FIELD_SUBJECT"), 'subject', $listDirn, $listOrder);?>
+						</th>
+						<th class="minwidth-150px">
+							<?php echo JHtml::_('grid.sort', Text::_("COM_TJNOTIFICATIONS_VIEW_NOTIFICATIONS_DEFAULT_FIELD_BODY"), 'title', $listDirn, $listOrder);?>
+						</th>
+						<th width="10%" class="">
+							<?php echo JHtml::_('grid.sort', Text::_("COM_TJNOTIFICATIONS_VIEW_NOTIFICATIONS_DEFAULT_FIELD_TO"), 'to', $listDirn, $listOrder);?>
+						</th>
+
+						<th width="10%" class="">
+							<?php echo JHtml::_('grid.sort', Text::_("COM_TJNOTIFICATIONS_VIEW_NOTIFICATIONS_DEFAULT_FIELD_CC"), 'cc', $listDirn, $listOrder);?>
+						</th>
+
+						<th width="10%" class="">
+							<?php echo JHtml::_('grid.sort', Text::_("COM_TJNOTIFICATIONS_VIEW_NOTIFICATIONS_DEFAULT_FIELD_BCC"), 'bcc', $listDirn, $listOrder);?>
+						</th>
+
+						<th width="10%" class="">
+							<?php echo JHtml::_('grid.sort', Text::_("COM_TJNOTIFICATIONS_VIEW_NOTIFICATIONS_DEFAULT_FIELD_DATE"), 'date', $listDirn, $listOrder);?>
+						</th>
+
+						<th width="" class="minwidth-150px">
+							<?php echo JHtml::_('grid.sort', Text::_("COM_TJNOTIFICATIONS_VIEW_NOTIFICATIONS_DEFAULT_FIELD_STATE"), 'state', $listDirn, $listOrder);?>
+						</th>
+
+						<th width="" class="">
+							<?php echo JHtml::_('grid.sort', Text::_("COM_TJNOTIFICATIONS_VIEW_NOTIFICATIONS_DEFAULT_FIELD_KEY"), 'key', $listDirn, $listOrder);?>
+						</th>
+
+						<th width="10%" class="">
+							<?php echo JHtml::_('grid.sort', Text::_("COM_TJNOTIFICATIONS_VIEW_NOTIFICATIONS_DEFAULT_FIELD_PROVIDER"), 'provider', $listDirn, $listOrder);?>
+						</th>
+
+						<th width="10%" class="">
+							<?php echo JHtml::_('grid.sort', Text::_("COM_TJNOTIFICATIONS_VIEW_NOTIFICATIONS_DEFAULT_FIELD_FROM"), 'from', $listDirn, $listOrder);?>
+						</th>
+						<th width="10%" class="">
+							<?php echo JHtml::_('grid.sort', Text::_("COM_TJNOTIFICATIONS_VIEW_NOTIFICATIONS_DEFAULT_FIELD_PARAMS"), 'params', $listDirn, $listOrder);?>
+						</th>
+						<th width="2%" class="">
+							<?php echo JHtml::_('grid.sort', Text::_("COM_TJNOTIFICATIONS_VIEW_NOTIFICATIONS_DEFAULT_FIELD_ID"), 'id', $listDirn, $listOrder);?>
+						</th>
+					</tr>
+					</thead>
+					<tbody>
+						<?php if (!empty($this->items)) : ?>
+							<?php foreach ($this->items as $i => $row) :
+								$link
+							?>
+								<tr>
+									<td class="">
+										<?php echo JHtml::_('grid.id', $i, $row->id); ?>
+									</td>
+									<td class="">
+											<?php echo htmlspecialchars($row->subject, ENT_COMPAT, 'UTF-8');
+									?>
+									</td>
+									<td class="">
+									<a id="modal_info" class="modal" href="<?php echo Route::_('index.php?option=com_tjnotifications&tmpl=component&view=logs&layout=body&id='. $row->id); ?>"><?php echo Text::_("COM_TJNOTIFICATIONS_TITLE_VIEW_CONTENTS");?></a>
```

Comment:
@punambaravkar Use class instead of id. We never use same HTML id multiple time in page.